spec/features/platform_admin_can_edit_vendor_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.feature "Platform admin can edit vendors" do
+  scenario "Platform admin and vendor see vendor's updated status" do
+    user = User.create(username: "Jones", password: "password", password_confirmation: "password", email: "jones@gmail.com")
+    user.roles.create(name: "platform_admin")
+
+    vendor = Vendor.create(name: "Jojo blu")
+
+    visit root_path
+    click_on "Login"
+    fill_in "Username", with: "Jones"
+    fill_in "Password", with: "password"
+    click_on "Sign In"
+
+    visit vendors_path
+
+    within(".vendor") do
+      expect(page).not_to have_content "Jojo blu"
+    end
+
+    visit dashboard_path(user.id)
+    click_on "All Vendors"
+    expect(current_path).to eq platform_admin_vendors_path
+    expect(page).to have_content "Jojo blu"
+
+    expect(page).to have_content "Status: Inactive"
+    click_on "Edit"
+    fill_in "Status", with: "Active"
+    click_on "Update Vendor"
```

Comment:
Agree that we should not be accepting this as text. Could also potentially use a dropdown.